main.asm

```diff
@@ -49,7 +49,10 @@ INCLUDE "engine/overworld/player_state.asm"
 INCLUDE "engine/events/poison.asm"
 INCLUDE "engine/overworld/tilesets.asm"
 INCLUDE "engine/overworld/daycare_exp.asm"
-INCLUDE "data/maps/hide_show_data.asm"
+
+
+SECTION "bank3 2", ROMX
```

Comment:
If a bank is going to be split into 1 and 2, it should probably have a more specific name (see the rest in layout.link). Not sure what though.

Reply:
Actually I'd prefer the solution of exporting these constants from maps.o, thus having them accessible in main.o.

Reply:
Yeah. We have #264 on the radar. But until that gets tackled, it shouldn't hold up this important update I'd say.

Reply:
> Actually I'd prefer the solution of exporting these constants from maps.o, thus having them accessible in main.o.

Yeah I'd be open for that too.

Seems tedious to add a whole bunch of export statements to all the data/maps/objects/ files. Would it be silly to make a macro like `const_export` or something?

Reply:
Yes, definitely use a macro so you only have to write `EXPORT` once. I thought they already had a unique macro, but yeah, if it's just `const` then `const_export` would work.

Maybe even just export the ones that actually get used outside?

Reply:
We could also possibly just combine the `object_const_def` lists into the `def_object_events` lists. They're already together in the same file and I'm pretty sure they're always the same length.

Reply:
> We could also possibly just combine the `object_const_def` lists into the `def_object_events` lists. They're already together in the same file and I'm pretty sure they're always the same length.

That sounds similar to https://github.com/pret/pokecrystal/issues/755. I think i'd opt to hold off on doing this just yet, personally.

Reply:
Updated